validator.php

```diff
@@ -33,6 +33,11 @@
     return true; // any other file gets checks and any other folder gets iterated
 };
 
+$isAcceptableVersionConstraint = function ($versionString) {
+    return (bool) preg_match('/^(\\<|\\>)(=){0,1}(\d+)(\.\d+)*(-(alpha|beta|rc)\d+){0,1}$/', $versionString);
```

Comment:
Strictly seen this would also allow something like `>=09` but that's probably negligible (we could of course also change the regex to something like `/^(\\<|\\>)(=){0,1}([1-9]\d*)(\.\d+)*(-(alpha|beta|rc)\d+){0,1}$/`).